lib/options.go

```diff
@@ -268,6 +268,9 @@ type Options struct {
 
 	// Discard Http Responses Body
 	DiscardResponseBodies null.Bool `json:"discardResponseBodies" envconfig:"discard_response_bodies"`
+
+	// Redirect console logging to a file
+	RedirectConsoleToFile null.String `json:"redirectConsoleToFile" envconfig:"redirect_console_to_file"`
```

Comment:
I'm not sure if this option should actually be accessible from the exported in-script `options`. It feels like we should disable it (maybe a `json:"-"` struct tag here will be enough?) for the same security and UX reasons (mostly with regards to distributed and cloud execution) we don't allow scripts to write stuff in files directly. 

I see this functionality as something that, for the moment at least, would be meant primarily to help with local debugging of k6 scripts. We can probably extend it in the future, when we properly implement distributed execution inside of k6 (https://github.com/loadimpact/k6/issues/140), but for now I think we should only allow configuring this via environment variables or CLI flags.